subsys/disk/disk_access_ram.c

```diff
@@ -20,10 +20,10 @@
  */
 #include "fat12_ramdisk.h"
 #else
-/* A 96KB RAM Disk, which meets ELM FAT fs's minimum block requirement. Fit for
```

Comment:
I don't think this comment should be deleted.
Is there a Kconfig which indicates the filesystem in use? Seems like if the size is made smaller than 96 the fs will be invalid

Reply:
As written in the commit message, I tested tests/posix/fs with 80K and it passed. So, we either have an fs test which doesn't detect invalid fs, we have another fs rather than fatfs by default, or that comment isn't exactly right. Can make bets which is it ;-)

Reply:
> As written in the commit message, I tested tests/posix/fs with 80K and it passed.

To prove the point, I cherry-picked that change from https://github.com/zephyrproject-rtos/zephyr/pull/12984, as it's going to hit on master soon anyway.